jobbergate-agent/jobbergate_agent/tasks.py

```diff
@@ -7,13 +7,21 @@
 from loguru import logger
 
 from jobbergate_agent.clients.cluster_api import backend_client
+from jobbergate_agent.internals.update import self_update_agent
 from jobbergate_agent.jobbergate.submit import submit_pending_jobs
 from jobbergate_agent.jobbergate.update import update_active_jobs
 from jobbergate_agent.settings import SETTINGS
 from jobbergate_agent.utils.logging import log_error, logger_wraps
 from jobbergate_agent.utils.scheduler import BaseScheduler, Job
 
 
+def self_update_task(scheduler: BaseScheduler) -> Job:
+    """
+    Schedule a task to self update the agent every ``TASK_SELF_UPDATE_INTERVAL_SECONDS`` seconds.
+    """
+    return scheduler.add_job(self_update_agent, "interval", seconds=SETTINGS.TASK_JOBS_INTERVAL_SECONDS)
```

Comment:
```suggestion
    return scheduler.add_job(self_update_agent, "interval", seconds=SETTINGS.TASK_SELF_UPDATE_INTERVAL_SECONDS)
```

A dedicated config would be useful for it. I see the docstring indicates one, but it is not reflected on the code here and on `settings.py`.

Reply:
see [7e288f1](https://github.com/omnivector-solutions/jobbergate/pull/518/commits/7e288f1dcece8e6321d4298875482e3092e04982)